scripts/package.js

```diff
@@ -0,0 +1,86 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const commander = require('commander');
+const fs = require('fs');
+const mkdirp = require('mkdirp');
+const path = require('path');
+const replace = require('replace-in-file');
+
+function stageFiles(commander, outputDir, datFile) {
+  const baseDatFile = path.parse(datFile).base;
+
+  const originalManifest = path.join('manifests', commander.type + '-manifest.json');
+  const updatedManifest = path.join(outputDir, 'manifest.json');
+
+  const replaceOptions = {
+    files: updatedManifest,
+    from: /0\.0\.0/g,
+    to: commander.setVersion,
+  };
+
+  mkdirp.sync(outputDir);
+
+  fs.copyFileSync(originalManifest, updatedManifest);
+  fs.copyFileSync(datFile, path.join(outputDir, baseDatFile));
+
+  replace.sync(replaceOptions);
```

Comment:
This returns a promise by looking in `node_modules/replace-in-file/lib/replace-in-file.js` when no callback is specified for the second param, so I'm pretty sure it will be picked up by the global `unhandledRejection` that I mentioned.

Reply:
I think you may be looking at the async version of `replace`? I don't think the synchronous version that I'm calling has any optional params, but I definitely hear you as far as installing the global handler - thanks for the tip!

Reply:
yep thanks